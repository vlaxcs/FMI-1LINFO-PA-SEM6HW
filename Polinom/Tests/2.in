-1 3 -6 -8 -3
2