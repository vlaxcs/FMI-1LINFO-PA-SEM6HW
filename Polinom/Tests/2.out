(-9)x^5 + (-8)x^4 + (-6)x^3 + (-3)x^2 + (-1)x^1 + (3)